### ISSUE TITLE
Create component library

### ISSUE THREAD

**Issue by jackbemi** [EMI-Advisors/C4BH#12]
Refactor code to pull commonly used components into seperate react components.
Things like buttons, tables, links, etc should have consistent styles. They can be overridden in some cases, but should be largely the same by default.
Also would be great to get a theme defined: a file to define a color pallete, which we can use to apply colors to elements.
should have main color, accent color, background color, main color tint, background color tint, etc.

Connect Figma to Amplify:
- [x] link Figma account 
- [x] import theme & components from Figma to Amplify studio
- [x] pull currently used components into their own files & apply Amplify theme 

Design components & theme for the webpage in Figma:
- [x] color and fonts
- [x] primitives (buttons, inputs, icons)
- [ ] table component or just a style might be good
- [x] reactive / resizing page layout #26 
- [x] sidebar with a list of headings and sections and stuff like a table of contents
- [x] footer
- [ ] (nice to have) style the form components to look more in line with the rest of the website's style

**Comment by NotAnotherDoorNob** [comment]
Was able to successfully connect figma to AWS amplify and can pull amplify UI updates into my local development space. Here is the Figma file where design system rules and components will be located going forward [https://www.figma.com/file/mOtMkkvxbiiyPrcW0exjdG/C4BH-AWS-Amplify-UI-Kit?type=design&node-id=3846%3A2689&mode=design&t=WXJNoKB1cMQfHweS-1]

**Comment by NotAnotherDoorNob** [comment]
Update* I re-did the header navigation to follow the amplify theme and pulled it into the amplify UI. We talked about implementing the CSS holy grail layout, I did a pass at a side nav as well which would allow us to do that. The nav will effect how the size of the body and the body's breakpoints so it'll be helpful for me to to nail that down before moving on. Hope to talk more next week- feel free to take a look in the meantime [https://www.figma.com/file/mOtMkkvxbiiyPrcW0exjdG/C4BH-AWS-Amplify-UI-Kit?type=design&node-id=4877%3A4610&mode=design&t=mvdNGYJVHjQlU4Si-1](url)
